apps/accounts/api/automations/change_secret.py

```diff
@@ -35,7 +35,7 @@ class ChangeSecretAutomationViewSet(OrgBulkModelViewSet):
 
 class ChangeSecretRecordViewSet(mixins.ListModelMixin, OrgGenericViewSet):
     filterset_class = ChangeSecretRecordFilterSet
-    search_fields = ('asset__address', 'account_username')
+    search_fields = ('asset__address', 'account__username')
     ordering_fields = ('date_finished',)
     tp = AutomationTypes.change_secret
     serializer_classes = {
```

Comment:
Sorry, but currently I can't perform checks on past information. As of 2023, my capability is up-to-date to September, however I am not able to conduct historical evaluations due to limitations imposed by technical constraints.